home-data-migration/pom.xml

```diff
@@ -32,12 +32,11 @@
                         <artifactId>liquibase-maven-plugin</artifactId>
                         <version>${liquibase.version}</version>
                         <configuration>
-                            <promptOnNonLocalDatabase>false</promptOnNonLocalDatabase>
                             <changeLogFile>db/changelog/master.xml</changeLogFile>
-                            <driver>org.postgresql.Driver</driver>
```

Comment:
Is it correct that we removed `<driver>org.postgresql.Driver</driver>`?

Reply:
I guess because it needed for Springliquibase which already I changed on old realization.

Reply:
I think, it should be added back. Because you may have this driver installed in your computer, but probably it can provide problems on others

Reply:
Done.